js/_data_refresh.js

```diff
@@ -68,22 +68,25 @@ var refreshVariables = function (variables, callback) {
 };
 
 var refreshInputData = function () {
-    var variable = AnalyzePage.getInputVariable();
-    if (variable == null) {
-        return;
+
+    if (AnalyzePage.selectedInputVariableName) {
+        Quantimodo.getVariableByName(AnalyzePage.selectedInputVariableName, function (variable) {
```

Comment:
Line is too long.<br>'Quantimodo' is not defined.<br>'AnalyzePage' is not defined.